docs/release-notes.md

```diff
@@ -1,230 +1,124 @@
-- **cli:** Add json flag to version command
-- **log:** adds -json option to all rpc commands
-- **output:** adds json output for amtinfo command
-- update password with user input
-- **pthi:** converts amt and pthi commands to go from C
-- see change log for full list of changes
-#### Sample Web UI
-- add-dialog: uses rpc-go syntax (#c729709)
-- cypress: consolidate http codes (#0ebdb3f)
-- see change log for full list of changes
-#### UI Toolkit
-- upgrade ws from 8.4.2 to 8.5.0 (#228166b)
-- **npm:** add job to publish to npm (#33bdf7e) 
-- **semantic-release:** adds automated releases (#fde186b)
-- see change log for full list of changes
-#### WSMAN-MESSAGES
-*Expect this set of libraries to version up very fast as we continue to enhance its capabilities to support the rest of the Open AMT Cloud Toolkit.  Since this is new, adding the full set of changes since 1.0.0.*
-
-v2.1.0 - 2022-03-15: 
-
-- **amt:** add wifiportconfiguration service PUT and GET calls (#b9767cf)
-
-v2.0.1 - 2022-03-02: 
-
-- **AMT:** update wifi_port naming convention ([#86](https://github.com/open-amt-cloud-toolkit/wsman-messages/issues/86)) (#4d2830d) 
+<p class="divider"></p>
+## Note From the Team
 
-v2.0.0 - 2022-03-02: 
+Hey everyone,
 
-- **tls-configuration:** add support for TLS Connection type (#0bcddb6) 
-- **BREAKING CHANGE:** messageID is no longer passed in to functions and is tracked internally
+It has been a very quick 6 weeks and the team is very excited to deliver Open AMT Cloud Toolkit version 2.3.  Unlike previous releases, every component of the Toolkit has been updated in some way.  If you haven't had a chance yet, I encourage you to watch the release video where Mike provides some highlights from this release.  Additionally, we've added a [Video Walkthroughs](https://open-amt-cloud-toolkit.github.io/docs/2.3/videos/) section to our documentation where Bryan Wendlandt has been creating video tutorials of our [Getting Started Guide](https://open-amt-cloud-toolkit.github.io/docs/2.3/GetStarted/prerequisites/).  Watch that space for additional video content to be delivered there.  Don't forget to like and subscribe!
 
-v1.7.0 - 2022-02-24
+Find out [what's new](#whats-new) and delve into the [details](#get-the-details) below-- and enjoy our new release of the toolkit.
 
-- **general-settings:** adds PUT method for AMT_GeneralSettings (#f39c0df) 
+*Best wishes,*<br>
+*Matt Primrose | Product Owner | The Open AMT Cloud Toolkit Team*
+<br>
+<p class="divider"></p>
 
-v1.6.0 - 2022-02-23
+## What's New?
 
-- **amt:** adds WiFiPortConfigurationService to amt. (#d2a2f0e) 
+:material-star:** Customer Request: RPC-Go as a library**
 
-v1.5.0 - 2022-02-15
+We have added an option to build RPC as a library, allowing developers to import RPC into their applications as a dll or shared object.  Instead of deploying and executing RPC as an application, developers can call and monitor RPC directly within their own application.
```

Comment:
gcc is still required for this, should probably mention it since in the section below we say its no longer required.

Reply:
Fixed